desktop/cmp/contextmenu/ContextMenu.js

```diff
@@ -49,7 +48,9 @@ ContextMenu.propTypes = {
 function parseMenuItems(items) {
     items = items.map(item => {
         if (item === '-' || isValidElement(item)) return item;
-
+        if (item.displayFn) {
+            item = {...item, ...item.displayFn()};
```

Comment:
this displayFn is a little different than the one used by RecordAction, which actually takes the existing item, and returns a new config.  That setup can be useful when trying to write a general display function.  Not sure what the right thing to do here is.